config/migrations/2023/20230215141257-set-up-climate-2-1-subsidy-for-specific-list-of-governments.sparql

```diff
@@ -0,0 +1,333 @@
+PREFIX mobiliteit:    <https://data.vlaanderen.be/ns/mobiliteit#>
+PREFIX lblodSubsidie: <http://lblod.data.gift/vocabularies/subsidie/>
+PREFIX dct:           <http://purl.org/dc/terms/>
+PREFIX subsidie:      <http://data.vlaanderen.be/ns/subsidie#>
+PREFIX gleif:         <https://www.gleif.org/ontology/Base/>
+PREFIX mu:            <http://mu.semte.ch/vocabularies/core/>
+PREFIX cpsv:          <http://purl.org/vocab/cpsv#>
+PREFIX common:        <http://www.w3.org/2007/uwa/context/common.owl#>
+PREFIX xkos:          <http://rdf-vocabulary.ddialliance.org/xkos#>
+PREFIX qb:            <http://purl.org/linked-data/cube#>
+PREFIX m8g:	          <http://data.europa.eu/m8g/>
+
+INSERT DATA {
+  GRAPH <http://mu.semte.ch/graphs/public> {
+     # Period of the Series
+    <http://data.lblod.info/id/periodes/46e05ff6-08f4-4ff9-a831-8897de75f25c>
+        a             m8g:PeriodOfTime ;
+        mu:uuid       "46e05ff6-08f4-4ff9-a831-8897de75f25c" ;
+        m8g:startTime "2023-01-01T03:00:00Z"^^xsd:dateTime ;
+        m8g:endTime   "2024-12-31T22:59:00Z"^^xsd:dateTime .
+
+    # Period of step Indienen pact
+    <http://data.lblod.info/id/periodes/33f16980-1d9b-49b9-b26c-d595ecb8689f>
+        a             m8g:PeriodOfTime ;
+        mu:uuid       "33f16980-1d9b-49b9-b26c-d595ecb8689f" ;
+        m8g:startTime "2023-02-15T03:00:00Z"^^xsd:dateTime ;
+        m8g:endTime   "2023-06-05T21:59:00Z"^^xsd:dateTime .
```

Comment:
Also, I since we only care about the time in Brussels TZ, wouldn't it be simpler to simply insert it like that?

`2023-06-05T23:59:00+02:00`

Reply:
There shouldn't be any difference I think with going the manual route. In both cases, we should be careful due to daylight saving times in order to avoid a date shifting to the following day, as what happened here. I think it's better to use the timezone version to be consistent with how subsidies have been added so far.